src/onegov/agency/cli.py

```diff
@@ -293,6 +294,19 @@ def _create_pdf(request, app):
     return _create_pdf
 
 
+@cli.command('export-xlsx')
+@pass_group_context
+@click.option('--people', default=True, is_flag=True)
+def export_xlsx(group_context, people):
+    def _export_xlsx(request, app):
```

Comment:
The underscore prefix is not really necessary here - nobody can reach this function anyway, so you don't need to signal that it is internal.

Reply:
I did this mainly because all other functions look the same and thus would like to leave it that way.